lib/md009.js

```diff
@@ -30,9 +30,11 @@ module.exports = {
       listItemLineNumbers.sort((a, b) => a - b);
     }
     const expected = (brSpaces < 2) ? 0 : brSpaces;
-    forEachLine(lineMetadata(), (line, lineIndex) => {
+    let inFencedCode = 0;
+    forEachLine(lineMetadata(), (line, lineIndex, inCode, onFence) => {
+      inFencedCode += onFence;
       const lineNumber = lineIndex + 1;
-      if (trailingSpaceRe.test(line) &&
+      if ((!inCode || inFencedCode !== 0) && trailingSpaceRe.test(line) &&
```

Comment:
Can we just compare to inFencedCode since non-0 values are truthy?

Reply:
We can. I prefer explicit to implicit, but it's your code 😄 . I'll update.